app/models/user.rb

```diff
@@ -1,3 +1,8 @@
 class User < ActiveRecord::Base
   has_many :microposts
+
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.